news-feed.css

```diff
@@ -0,0 +1,31 @@
+.news-feed {
+  display: grid;
+  grid-template-rows: 0.3fr 1fr 1fr;
+  border-left: solid pink;
+}
+.search-bar {
+  height: auto;
+  background-color: rgb(211, 223, 223);
+  gap: 10px;
+  margin-top: 5ch;
+  margin-bottom: 2ch;
+  margin-left: 10px;
+  margin-right: 10px;
+  border-radius: 10px;
+  text-align: center;
+}
+.search-bar img {
+  width: 3ch;
+
```

Comment:
You could move the search bar icon to the left if you made the img `position:absolute` and its container `position:relative`